bbc/simorgh
# [WSTEAM1-253] ATI Tracking: Add categoryName (x17) tagging to Optimo Article Pages

Resolves #[253](https://jira.dev.bbc.co.uk/browse/WSTEAM1-253)

**Overall change:**
Adds ATI tracking for category name tagging to Optimo Article Pages (already exists on CPS)

**Code changes:**
- Use `getThingAttributes` to get `thingEnglishLabel` (tagging) from articleData and add value(s) to new `categoryName` field
- Update Tests
- Update Snapshots

**Testing:**

- Go to: http://localhost:7080/kyrgyz/articles/c88ld5g4xxxo
- Filter by `idclient`
- Click on the first result and then check within the `Network > Headers` tab that there are values in the x17 field
